controllers/operands/kubevirt.go

```diff
@@ -669,13 +669,21 @@ func hcTLSSecurityProfileToKv(profile *openshiftconfigv1.TLSSecurityProfile) *ku
 	}
 }
 
+func hcMemoryOvercommitToKv(hwd *hcov1beta1.HigherWorkloadDensityConfiguration) int {
+	return hwd.MemoryOvercommitPercentage
+}
+
```

Comment:
Why is this needed? 

usually we need translation functions when KV type is not the same as HCO type, but here both of them are `int`.